applications/machine_learning/boards/thingy53.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <init.h>
+
+#include <usb/usb_device.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(thingy53_setup);
+
+
+static int usb_cdc_init(const struct device *dev)
+{
+	int err = usb_enable(NULL);
+
+	if (err) {
+		LOG_ERR("Failed to enable USB");
```

Comment:
What is the point of printing error log message that USB enable failed if logs are printed through USB, so I guess it won't be displayed? Or am I missing something?

Reply:
Yes, you are right. Im currently reworking this a bit. This file won't be existing but will be in other place. I will remove this log from there.